src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,40 @@
 using namespace std;
 
 int t06_cinema() {
-    
+    int n, m, k, str = 0;
+	cin >> n >> m;
+    int arr[20][20];
+
+	for(int i = 0; i < n; i++)
+        for(int j = 0; j < m; j++)
+            cin >> arr[i][j];
+
+    cin >> k;
+
+    int cou = 0;
+    bool is = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'is' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4695003)